jaxley/utils/cell_utils.py

```diff
@@ -128,8 +128,9 @@ def compute_parents_in_level(levels, par_inds, parents_row_and_col):
     level_of_parent = levels[par_inds]
     parents_in_each_level = []
     for l in range(np.max(levels)):
-        parents_inds_in_current_level = jnp.where(level_of_parent == l)[0]
+        parents_inds_in_current_level = np.where(level_of_parent == l)[0]
```

Comment:
yes, there is a reason. After this change, every level has a different number of `nseg`. Therefore, anything that influences the `nseg` (of every level) has to be static at runtime. The easiest way to enforce this is to have all those things as numpy arrays (instead of jax arrays which are traced)

Reply:
Ah makes sense! Thanks for the clarification.